sample-lnd.conf

```diff
@@ -619,8 +619,12 @@ litecoin.node=ltcd
 ; probability (default: 1h0m0s)
 ; routerrpc.penaltyhalflife=2h                            
 
-; The (virtual) cost in sats of a failed payment attempt (default: 100)
-; routerrpc.attemptcost=90                                
+; The (virtual) fixed cost in sats of a failed payment attempt (default: 100)
+; routerrpc.attemptcost=90     
+
+; The (virtual) proportional cost in ppm of the total amount of a failed payment
+; attempt (default: 1000)
+; routerrpc.attemptcostppm=900
```

Comment:
should this match the default value?

Reply:
It seems that we use other values than the default in this file. Probably to demonstrate what it looks like with a non-default value.